api/net/Areas/Subscriber/Controllers/ReportInstanceController.cs

```diff
@@ -276,5 +276,30 @@ public FileResult GenerateExcel(int id)
 
         return File(bytes, "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet");
     }
+
+    /// <summary>
+    /// Find report instance for the specified report and date.
+    /// </summary>
+    /// <param name="reportId"></param>
+    /// <param name="publishedOn"></param>
+    /// <returns></returns>
+    [HttpGet("{reportId}/report")]
+    [Produces(MediaTypeNames.Application.Json)]
+    [ProducesResponseType(typeof(ReportInstanceModel), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
+    [SwaggerOperation(Tags = new[] { "ReportInstance" })]
+    public IActionResult FindByReportIdAndDate(int reportId, [FromQuery]DateTime publishedOn)
```

Comment:
Can we merge the existing `FindById` endpoint with this one?  It's only a query parameter difference.